src/applications/vaos/components/UpdateAddressAlert.jsx

```diff
@@ -1,31 +1,29 @@
 import React from 'react';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
-export default function UpdateAddressAlert({ address, showAlert, onHide }) {
-  const regexp = /^PO Box/;
-  if (showAlert && (!address || address.match(regexp))) {
```

Comment:
Moved this conditional into the `TypeOfCarePage` component so we can trigger the GA event if necessary in `componentDidMount()`